### ISSUE TITLE
As a data provider, I would prefer to have a single interface (input mask) for providing my metadata in case of a low level data ingest (instead of different interfaces for each object)

### ISSUE THREAD

**Issue by AndyDaniel1** [dzhw/metadatamanagement#2957]
We need an interface that contains all relevant fields for low-level metadata ingest (self-service). 

The interface should include all information for 

- data package 
- survey (optional?)
- instruments 
- data sets  

These elements should represent sections of the interface. The new interface of da|ra ist a good example for this kind of structured  interface. 

This is issue relates to #2911 since some information will move from data package to survey. If survey would be optional in this interface this information would also be optional (which is not possible). 

This issue also relates to #1707, as a save option should be mandatory for the interface discussed in this issue.

**Comment by tilovillwock** [comment]
To get this going, we need to clarify some things in order to provide a mockup of what that interface would look like.

**1. What is the starting point for a unified input mask going to be?**
As of now the workflow is: Click on "New" for a specific type > Fill in the data > Click on "Save <artifact>" to complete the action. Do we provide both workflows, the afore mentioned individual actions **AND** a unified input mask, or do we replace the individual actions?

**2. Could you give an example of what the da|ra interface looks like?**
For now it's conceivable that all types will simply be structured as collapsable sections with buttons to add and remove artifacts. Each section will use the existing input mask for each category.

**3. Is the unified input mask limited to the mentioned types only?**
Since we're probably ending up with collapsable sections we could also include the other artifact types. Otherwise we need to make sure that we're communicating clearly that the unified approach only includes certain types of artifacts.

**Misc**
* Regarding #1707: we probably need to introduce a temporary flag for datasets to make this work
* Regarding #2911: I think we could work on this in parallel, particularly moving fields from one form component to another, since we plan to reuse existing components in the unified form view anyway

@AndyDaniel1

**Comment by AndyDaniel1** [comment]
1 & 3) The goal is to have a unified mask for the mentioned objects as publications, variables and questions are imported via csv / json import and concepts (as publications) are overarching objects. The individual solution for all integrated objects would be obsolete after the implementation of a unified interface.   

2)  I have attached a .pdf with a mock-up that also includes the ideas from #1707.

For now I would postpone the work on #2911 as it might have to many dependencies. I will think about it until our meeting on Wednesday.  

[2957.pdf](https://github.com/dzhw/metadatamanagement/files/13073081/2957.pdf)
